charts/revere/templates/job.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.name }}-job
```

Comment:
I think this could also be done via an `initContainer` rather than an entire separate job that spins up new pod. No big deal though, just another option.

Reply:
actually very true, ive just written more job yamls than initContainer yamls 🤷  this is an artifact from when we were thinking we were going to have multiple revere replicas--then a job to only run `prepare` once makes more sense. I'm going to leave it as-is for now as one less thing to change if we move to multiple replicas in the future